src/OrbitDB.js

```diff
@@ -239,7 +259,7 @@ class OrbitDB {
     // this is what hooks us into the message propagation layer
     // and the p2p network
     if (opts.replicate && this._pubsub) { await this._pubsub.subscribe(addr, this._onMessage.bind(this), this._onPeerConnected.bind(this)) }
-
+    this.events.emit('open', store.address.toString())
```

Comment:
I feel like this is the meat of the pr and allows for users to do everything else with minimal logic.

Reply:
Yeah, true. Mostly I want just these three events, `open`, `load`, and `ready`. `ready` comes from the db scope so it's better imo to just add the facility to hoist any and all db scope events to the parent instance scope